### ISSUE TITLE
"java.lang.IllegalStateException: Attempt to cancel when process is still transitioning"

### ISSUE THREAD

**Issue by donnlee** [rancher/rancher#5606]
I have a single-container service ('grafana/grafana') that seems to be wedged in state "Starting (Scheduling)".
I tried stopping and starting the container (and service) in Rancher webUI.

Also tried restarting rancher-server:
```
docker stop <id_of_rancher-server>
docker start <id_of_rancher-server>
```

"View in API" (for container):
```
{
"id": "1i936",
"type": "container",
```

rancher-server log:

```
2016-08-01 18:34:00,647 ERROR [d34c8d9a-9a10-4560-8fd7-1b04cda130c9:131925] [instance:936] [instance.start] []
[erviceReplay-10] [c.p.e.p.i.DefaultProcessInstanceImpl] Unknown exception java.lang.IllegalStateException: Att
empt to cancel when process is still transitioning
        at io.cattle.platform.engine.process.impl.DefaultProcessInstanceImpl.runDelegateLoop(DefaultProcessInst
anceImpl.java:190) [cattle-framework-engine-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.engine.process.impl.DefaultProcessInstanceImpl.executeWithProcessInstanceLock(Def
aultProcessInstanceImpl.java:157) [cattle-framework-engine-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.engine.process.impl.DefaultProcessInstanceImpl$1.doWithLock(DefaultProcessInstanc
eImpl.java:107) [cattle-framework-engine-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.engine.process.impl.DefaultProcessInstanceImpl$1.doWithLock(DefaultProcessInstanc
eImpl.java:104) [cattle-framework-engine-0.5.0-SNAPSHOT.jar:na]
        at io.cattle.platform.lock.impl.AbstractLockManagerImpl$3.doWithLock(AbstractLockManagerImpl.java:40) [
cattle-framework-lock-0.5.0-SNAPSHOT.jar:na]
<...>
```

Health check for this service:

```
grafana:
  scale: 1
  health_check:
    port: 3000
    interval: 3000
    initializing_timeout: 60000
    unhealthy_threshold: 3
    strategy: recreate
    request_line: GET "/" "HTTP/1.0"
    healthy_threshold: 2
    response_timeout: 2000
```

What else should I do?  Other info I should provide?

---
| Useful | Info |
| :-- | :-- |
|Versions|Rancher `v1.1.1` Cattle: `v0.165.7` UI: `v1.1.12` |
|Access|`Disabled`|
|Orchestration|`Cattle`|
|Route|`container.labels`|

**Comment by deniseschannon** [comment]
What's the API for the service?

Are you still in this situation for this service?

**Comment by donnlee** [comment]
@deniseschannon thank you for the reply.  This was http over tcp port 3000. 
Unfortunately, the VM environment I was using for this test has been torn-down.  I left it running for about 2 weeks, and nothing changed.  I didn't know how to diagnose further so I gave up.
Interestingly enough, this is the 3rd time I've had a Rancher environment decay over a few days to the point it gets wedged (containers in "Starting..." or similar state) and I could not revive it.
Each time I have to destroy all the hosts (VMs) and rebuild the stacks/services from scratch.

I figured I would try to recreate my environment in Kubernetes + CoreOS + flannel and then return later to Rancher, and hope that whatever problem I ran into is fixed in a future Rancher release.

If I encounter the error again, I'm open a new issue.  Thanks
